esmvaltool/recipes/recipe_climwip_brunner2019_med.yml

```diff
@@ -0,0 +1,295 @@
+# ESMValTool
+# recipe_climwip_brunner2019.yml
+---
+documentation:
+  description: >
+    Calculate weights similar to Brunner et al. (2019).
+    Changes compared to Brunner et al. (2019):
+      - Use model native land-sea mask (instead of regionmask)
+      - Use ESMValCore extract_shape to cut regions (instead of regionmask)
+      - The models CCSM4 and MIROC5 (r1) had to be excluded due to errors in the ESMValCore pre-processor
+      - Use ERA5 instead of three older reanalyses as observational data set
+    This recipe is intended as a template - it calculates weights for the Mediterranean (MED) SREX
+    region. Small adaptation are needed to calculate the other regions (see documentation for guidance).
+  authors:
+    - brunner_lukas
+    - lorenz_ruth
+  maintainer:
+    - brunner_lukas
+  references:
+    - brunner2019
+
+datasets: &model_data
+  - {dataset: ACCESS1-0, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: ACCESS1-3, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: bcc-csm1-1-m, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: bcc-csm1-1, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: BNU-ESM, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: r1i1p1}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: r2i1p1}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: r3i1p1}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: r4i1p1}
+  - {dataset: CCSM4, project: CMIP5, exp: [historical, rcp85], ensemble: r5i1p1}
+  # pre-processor can't merge historical and RCP85 - unclear why
```

Comment:
Have you checked for missing data/overlap between historical and future/missing ensemble member for either one?
What is the error message you get?

Reply:
Why do you not use the short version for the ensemble members? e.g. ensemble: "r(1:5)i1p1"

Reply:
@Peter9192 : I will check again, but as far as I can remember there was no obvious reason why it failed for that model...
@ruthlorenz : I think that was the recipe filler?

Reply:
Here is the logfile (the debug file is too large for upload):

[main_log.txt](https://github.com/ESMValGroup/ESMValTool/files/6279587/main_log.txt)

I'm okay with just excluding this member though